packages/medulas-react-components/src/theme/utils/mui.ts

```diff
@@ -142,6 +142,29 @@ const themeObject: ThemeOptions = {
         fontSize: '1.4rem',
       },
     },
+    MuiSnackbarContent: {
+      root: {
+        [theme.breakpoints.up('xs')]: {
+          borderRadius: 2,
+          boxShadow: '0 0 6px 0 #f3f4fb',
+        },
+        width: 460,
```

Comment:
I believe the chrome extension only has... 350px as width, is this a problem? can you paste a couple of screenshots in the PR's description?